src/ResourceManager/ManagementGroups/Commands.ManagementGroups/Models/PSManagementGroupNoChildren.cs

```diff
@@ -0,0 +1,71 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.ManagementGroups.Models;
+
+namespace Microsoft.Azure.Commands.ManagementGroups.Models
+{
+    public class PSManagementGroupNoChildren
```

Comment:
@cormacpayne We can have a single class by changing the formatting file. However, rather than making the Lazy calls, we would prefer adding a note when Get is called (without `-Expand` or `-Recurse`) saying that the user needs to pass `-Expand` or `-Recurse` to access the children. This is because of three reasons - 
1) We need to release preview as soon as possible to our public preview customers. 
2) By API Design, Get is not meant to show children without `-Expand` and `-Recurse` 
3) I think we have already made all the changes needed for the preview as suggested in the review meeting. This change is something that can be aded in the future. Also, our new Powershell design is already in progress, which will provide the users which such flexibilty eventually.

I will be making the other changes. Please confirm if I can proceed with having one class by changing the formatting file and adding a note.